src/DonationForms/Migrations/CleanMultipleSlashesOnDB.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Give\DonationForms\Migrations;
+
+use Give\DonationForms\Actions\SanitizeDonationFormPreviewRequest;
+use Give\DonationForms\Models\DonationForm;
+use Give\DonationForms\Properties\FormSettings;
+use Give\Framework\Blocks\BlockCollection;
+use Give\Framework\Exceptions\Primitives\Exception;
+use Give\Framework\Migrations\Contracts\Migration;
+
+/**
+ * @unreleased
+ */
+class CleanMultipleSlashesOnDB extends Migration
+{
+
+    /**
+     * @unreleased
+     *
+     * @throws Exception
+     */
+    public function run()
+    {
+        global $wpdb;
+
+        $formMetaTable = $wpdb->prefix . 'give_formmeta';
+
+        $sql = $wpdb->prepare("SELECT form_id FROM $formMetaTable as fm WHERE
+        (fm.meta_key = 'formBuilderSettings' AND fm.meta_value LIKE '%\\\\\\\\\\\\\\\\\\\\\\\\%')
+        OR
+        (fm.meta_key = 'formBuilderFields' AND fm.meta_value LIKE '%\\\\\\\\\\\\\\\\\\\\\\\\%')");
```

Comment:
Is this a problem in the formBuilderFields meta as well? I thought it was only in the settings.

Reply:
I have the impression that I have seen the problem in the `formBuilderFields` before, but I can't replicate it. 

But maybe is just my brain is lying to me and I only added it on migration because this is also handled in the `SanitizeDonationFormPreviewRequest` class when the form builder loads as you can see on the attached screenshot. 

Not sure if we should keep it "just in case" or if is best to remove it.

![image](https://github.com/impress-org/givewp/assets/16308864/05e287a6-562c-4813-b977-7200e7987670)

Reply:
Since migrations directly affect customer data, I think it's important to keep the scope as small as possible and avoid speculating a problem.

We want to make sure our solution solves a specific problem and only touches what it needs.  If there is no evidence or possibility of this affecting `formBuilderFields` than my vote is _do not_ include it.